index.bs

```diff
@@ -979,15 +979,18 @@ The <dfn method for=FileSystemDirectoryHandle>getDirectoryHandle(|name|, |option
      [=FileSystemHandle/locator=] contains a file named |name|, or an empty
      directory named |name|, this will attempt to delete that file or directory.
 
-     Attempting to delete a file or directory that does not exist is considered success,
-     while attempting to delete a non-empty directory will result in a promise rejection.
+     Attempting to delete a file or directory that does not exist results in a promise
+     being rejected with a "NotFoundError" DOMException. Attempting to delete a non-empty
+     directory results in a promise being rejected with an "InvalidModificationError"
+     DOMException.
 
   : await |directoryHandle| . {{FileSystemDirectoryHandle/removeEntry()|removeEntry}}(|name|, { {{FileSystemRemoveOptions/recursive}}: true })
   :: Removes the [=/file system entry=] named |name| in the [=directory entry=]
      [=locate an entry|locatable=] by |directoryHandle|'s [=FileSystemHandle/locator=].
      If that entry is a directory, its contents will also be deleted recursively.
 
-     Attempting to delete a file or directory that does not exist is considered success.
+     Attempting to delete a file or directory that does not exist results in a promise
+     being rejected with a "NotFoundError" DOMException.
```

Comment:
This change seems to presuppose an outcome to #11. The normative text doesn't seem to currently reject at all.

Reply:
Thank you for taking a first look @annevk! I'll pick this back up after the holidays. Will work with @diekus to get the Participant Data updated in the meantime.